src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -106,6 +107,11 @@ public static class ScriptConstants
         public const string LinuxMetricEventStreamName = "MS_FUNCTION_METRICS";
         public const string LinuxFunctionDetailsEventStreamName = "MS_FUNCTION_DETAILS";
 
+        public const string RunFromZipSettingName = "WEBSITE_RUN_FROM_ZIP";
+        public const string RunFromZipAltSettingName = "WEBSITE_USE_ZIP";
+        public const string ContainerGroupName = "CONTAINER_NAME";
```

Comment:
I already have this defined in EnvironmentSettingNames. Merge your new env vars into there